packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -179,6 +180,7 @@ class DescopeWc extends BaseDescopeWc {
         ? {
             callbackUrl: redirectAuthCallbackUrl,
             codeChallenge: redirectAuthCodeChallenge,
+            backupCallbackUri: redirectAuthBackupCallbackUri,
```

Comment:
is there is a chance that `redirectAuthBackupCallbackUri` exists and `redirectAuthCallbackUrl` does not exists?

![image](https://github.com/descope/descope-js/assets/10514677/cd22aa1c-ca5c-4966-9a72-4164cfd5d1f6)

Reply:
Good question - It's not allowed